src/ancillary/__tests__/anchor-request-params-parser.test.ts

```diff
@@ -11,8 +11,7 @@ import { bases } from 'multiformats/basics'
 import { GenesisFields } from '../../models/metadata.js'
 import { asDIDString } from '../did-string.js'
 import { mockRequest } from '../../controllers/__tests__/mock-request.util.js'
-import * as t from 'io-ts'
-import { isRight, Right } from 'fp-ts/lib/Either.js'
+import * as t from 'codeco'
```

Comment:
Although t appeared some places before, it's used a lot more now in tests. Can we name this something else, so it's a little easier to grok when its used in tests?

Reply:
As `t` effectively is a namespace, the name should be short (it appears in `t.type({a: t.string})`) and have low probability of conflicting with other code. `t` was chosen as it is traditionally what `io-ts` recommends. Capital `C`? Or `CDC`? Suggestions are welcome.